xla/service/hlo_runner_interface.h

```diff
@@ -214,11 +214,16 @@ class HloRunnerInterface {
   static absl::StatusOr<std::unique_ptr<HloModule>> CreateModuleFromString(
       absl::string_view hlo_string, const DebugOptions& debug_options);
 
+  // Creates an HloModule from the given proto.
+  static absl::StatusOr<std::unique_ptr<HloModule>> CreateModuleFromProto(
+      const HloModuleProto& proto, const DebugOptions& = {});
```

Comment:
Sorry, I missed this one.

```suggestion
      const HloModuleProto& proto,
      const DebugOptions& debug_options = DebugOptions::default_instance());
```